examples/Node/file.js

```diff
@@ -21,8 +21,8 @@ detector.on('silence', function () {
   console.log('silence');
 });
 
-detector.on('sound', function () {
-  console.log('sound');
+detector.on('sound', function (buffer) {
+  console.log('sound', buffer);
```

Comment:
You are right, as it's an example, I left the console.log for demo... If I remember well, it's not showing binary, but object reference ( Maybe it depends on node versions )
Anyway, it's not good to leave console.log ;-)